### ISSUE TITLE
Don't express route's travel time in days

### ISSUE THREAD

**Issue by verhovsky** [organicmaps/organicmaps#9000]
If you create a long route it will say that it will take "1 day, 2 hours (2033km)". It should just say "26 hours (2033km)". It should always cap out at "X hours" and never say days because driving for 24 hours takes more than 24 hours.

**Comment by biodranik** [comment]
For many, it's easier to pre-calculate longer routes in days, not in hours. Try to guess quickly how many days a 69-hour trip may take. I did not understand the argument about driving for 24 hours.

**Comment by verhovsky** [comment]
Because "1 day" of driving means at most like 16 hours of driving because you need to sleep and take breaks, whereas saying "24 hours" of driving means exactly 24 hours of just driving.

Google Maps also works this way.

**Comment by Maschga** [comment]
How about adding a setting for this use case? It is comparable to the "Units of measurement" setting.

**Comment by verhovsky** [comment]
No, it's such a minor thing and most users will never create a route that long that it would be a useless setting.

**Comment by biodranik** [comment]
Google shows days at the bottom:
![image](https://github.com/user-attachments/assets/8413c5c6-e785-4506-b9f7-c1f0e613a68f)

**Comment by verhovsky** [comment]
but not at the top

![image](https://github.com/user-attachments/assets/4e72ff17-814f-47fc-aefc-72a9e1782726)

which is a smart way to solve it, have both ways written in different places.

Sorry, I was looking at the desktop where they have just hours.
